testutils/utils.go

```diff
@@ -274,6 +275,13 @@ func testAtomicPutCreate(t *testing.T, kv store.Store) {
 	assert.True(t, success)
 }
 
+func testAtomicPutWithSlashSuffuxKey(t *testing.T, kv store.Store) {
+	k1 := "testAtomicPutWithSlashSuffuxKey/key/"
```

Comment:
Oh! Forgot but this key should be cleaned up on the teardown function as it is failing on the second round of tests :)